src/Client/AmplitudeClient.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Client;
+
+use Consolidation\Config\Config;
+use Symfony\Component\Yaml\Yaml;
+use Zumba\Amplitude\Amplitude;
+
+/**
+ * Provides an amplitude client to log events.
+ */
+class AmplitudeClient {
+
+  /**
+   * Config directory to load Amplitude API Key YAML.
+   *
+   * @var array
+   */
+  protected $apiKeyDir = [
+    '/config',
+    'amplitude_api_key.yml',
+  ];
+
+  /**
+   * Amplitude API Key.
+   *
+   * @var string
+   */
+  protected $amplitudeApiKey = '';
+
+  /**
+   * The Zumba Amplitude client.
+   *
+   * @var \Zumba\Amplitude\Amplitude
+   */
+  protected $amplitude;
+
+  /**
+   * Constructs an instance of Amplitude Client.
+   *
+   * @param string $amplitude_user_id
+   *   The Amplitude user ID, usually the Content Hub Service
+   *   Client UUID which will be specific to subscription.
+   */
+  public function __construct(string $amplitude_user_id) {
+    $this->loadAmplitudeApiConfig();
+    $this->amplitude = (new Amplitude())->init($this->amplitudeApiKey, $amplitude_user_id);
+  }
+
+  /**
+   * Logs an event.
+   *
+   * @param string $name
+   *   The event name, e.g., "Fixture created".
+   * @param array $properties
+   *   An associative array of key/value pairs corresponding to properties or
+   *   attributes of the event.
+   *
+   * @see https://help.amplitude.com/hc/en-us/articles/115000465251#how-should-i-name-my-events
+   * @see https://help.amplitude.com/hc/en-us/articles/115000465251#event-properties
+   */
+  public function logEvent(string $name, array $properties = []): void {
+    if (!$this->amplitude) {
+      return;
+    }
+    $properties['timestamp'] = time();
+    $this->amplitude->logEvent($name, $properties);
+  }
+
+  /**
+   * Sets the amplitude key from the configuration yaml.
+   *
+   * @throws \Exception
+   */
+  private function loadAmplitudeApiConfig(): void {
```

Comment:
If I understand well we can get rid off this function if we hardcode the API key.